zerver/openapi/zulip.yaml

```diff
@@ -10582,6 +10587,13 @@ paths:
                           Whether notification emails in this organization are allowed to
                           contain Zulip the message content, or simply indicate that a new
                           message was sent.
+                      realm_enable_spectator_access:
+                        type: boolean
+                        description: |
+                          Present if `realm` is present in `fetch_event_types`.
+
+                          Whether to allow users to access web public streams without login.
+                          This setting also controls API access of web public streams.
```

Comment:
Extended this documentation and also added to the API changelog.  All changes to the API (i.e. every edit to this file) need an API changelog entry and feature level bump; our mobile and terminal developers rely on our doing that to be able to correctly implement features.